decidim-meetings/app/forms/decidim/meetings/admin/meeting_form.rb

```diff
@@ -3,6 +3,15 @@ module Decidim
   module Meetings
     module Admin
       # This class holds a Form to create/update meetings from Decidim's admin panel.
+      # Custom Virtus value to parse a String representing a Time using
```

Comment:
Docs are mixed and real class does not have docs directly. Maybe move line 5 before class definition?